src/lib/crypto/kimchi_bindings/stubs/src/arkworks/bigint_256.rs

```diff
@@ -239,12 +239,12 @@ mod tests {
     #[test]
     fn biguint() {
         let x = 10000.to_biguint().unwrap();
-        println!("biguint.to_string: {}", x.to_string());
+        println!("biguint.to_string: {}", x);
```

Comment:
At some point, `clippy` wanted me to write this with string interpolation, like so.
```
        println!("biguint.to_string: {x}");
```

Reply:
Yes, but here it didn't recommend it. Clippy seems happy like this :shrug: